sdk_container/src/third_party/coreos-overlay/profiles/coreos/base/package.accept_keywords

```diff
@@ -83,3 +83,7 @@
 =sys-libs/libsemanage-3.7 ~arm64
 =sys-libs/ncurses-6.5_p20250125 ~arm64
 =sys-process/audit-4.0.2-r1 ~arm64
+
+# enable NVIDIA drivers for ARM
+=x11-drivers/nvidia-drivers-550.144.03 ~arm64
+=x11-drivers/nvidia-drivers-535.230.02 ~arm64
```

Comment:
So slight complication here... I believe only 570 builds for arm64 with the kernel+gcc combination we're using - at least that's what I found when adding support for it to nvidia drivers recently.

How about disabling nvidia sysexts for arm64 for the time being, to focus on getting amd64 in shape.

Reply:
Ah, right. I was wondering why the ARM builds always fail with compilation errors. I'll disable it for now, which reminds me there currently isn't a way to make a sysext for AMD only. I might need to add another field to the sysext specifier..